doc-redirects.properties

```diff
@@ -27,6 +27,11 @@
 /docs/ref/command/*=/docs/latest/reference/command/
 /docs/ref/feature/*=/docs/latest/reference/feature/
 
+# Redirects for a file name change to runnable-jar-files.html. 20.0.0.8 was the latest version with the old file name.
+/docs/runnablejarfiles.html=/docs/runnable-jar-files.html.html
```

Comment:
this would be
`/docs/ref/general/runnablejarfiles.html=/docs/runnable-jar-files.html.html`

Reply:
The runnable jar files topic is in the ROOT module, not reference like the feature and server overviews. Did it still need `/ref` in the URL? I can't quite remember how the URL structure has evolved so just want to make sure.

Reply:
prior to antora it was at `/docs/ref/general/runnablejarfiles.html` 
it was never at `/docs/runnablejarfiles.html`

I don't expect this one to matter much as I can't imagine anyone having a bookmark to that after 1+ years and to my knowledge nothing references that old url.  However, doesn't hurt to have.

Reply:
https://web.archive.org/web/20191013144912/https://openliberty.io/docs/ref/general/

Reply:
thanks- updated